enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Notebook.tsx

```diff
@@ -1,33 +1,53 @@
 import { useMemo } from "react";
 
-import { useInteractiveQuestionContext } from "embedding-sdk/components/public/InteractiveQuestion/context";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
 import { useSelector } from "metabase/lib/redux";
 import { default as QBNotebook } from "metabase/query_builder/components/notebook/Notebook";
 import {
   isQuestionDirty,
   isQuestionRunnable,
 } from "metabase/query_builder/utils/question";
+import { getMetadata } from "metabase/selectors/metadata";
 import { getSetting } from "metabase/selectors/settings";
 import { ScrollArea } from "metabase/ui";
+import { sourceTableOrCardId } from "metabase-lib";
+import type Question from "metabase-lib/v1/Question";
 
 type NotebookProps = { onApply?: () => void };
 
 export const Notebook = ({ onApply = () => {} }: NotebookProps) => {
   const { question, originalQuestion, updateQuestion, runQuestion } =
     useInteractiveQuestionContext();
+  const metadata = useSelector(getMetadata);
 
-  const isDirty = useMemo(() => {
-    return isQuestionDirty(question, originalQuestion);
-  }, [question, originalQuestion]);
+  const isDirty = useMemo(
+    () => isQuestionDirty(question, originalQuestion),
+    [question, originalQuestion],
+  );
```

Comment:
nit: by removing the braces around the function body this 3 LOC block became 4 LOC block :)

Reply:
lol